Login/login.js

```diff
@@ -0,0 +1,61 @@
+window.onload = ()=> {
+    this.sessionStorage.setItem("username", "api-client");
+    this.sessionStorage.setItem("password", "KEV9EwC5SEvk4dF");
+}
+
+var input = document.getElementsByTagName('input');
+var login = document.getElementById('log-in');
+var form = document.querySelector('form');
+form.onsubmit = ()=>{return false}
+
+login.onclick = ()=>{
+
+    if ((input[0].value != "") && (input[1].value != ""))
+    {
+        if ((input[0].value == sessionStorage.getItem("username")) && (input[1].value 
+            == sessionStorage.getItem("password"))) 
```

Comment:
This code is too complicated.

The flow should look like this:
- get the values from the input elements
- if the username input value **is empty** **or** the password input value **is empty** 
  - then add an error
  - return
- if the username input value **and** the password input value **are not empty**
  - store the username and password in the session storage
  - return
And that's it

Reply:
So you don't need to compare usernames and passwords